### ISSUE TITLE
Feature: generate feeds from bookmarks

### ISSUE THREAD

**Issue by Menti** [plainmade/unmark#113]
This is a niche request. There are some uses of RSS feeds generated by a bookmark app:

- Automating the posting of links to Twitter, etc.
- Populating a blog sidebar with recent bookmarks.
- Subscribing to someone's bookmarks.

Some bookmarking apps generate RSS for the user's bookmarks, tags, etc. RSS would be the simplest and most generic way of delivering Unmark content to other services (readers, ifttt, a mobile app, etc.). Unmark could generate feeds for:

- Timeline
- Each label
- Each tag
- Ideally, even search.

Thanks for this great software!

**Comment by cdevroe** [comment]
@Menti I agree. I'm a huge fan of RSS personally. I think this is a must. Though we'll have to carefully plan it out since everyone's bookmarks are "password protected" even locally. So RSS may need to be optionally un-protected as well.

**Comment by Menti** [comment]
¡Thanks! I understand the difficulties and I'm sure you'l find some neat 
way to sort it out.

**Comment by heubergen** [comment]
+1

**Comment by cdevroe** [comment]
@Menti @heubergen We likely won't get to this within the next few builds. But if either of you would like to submit a PR with some options on how to tackle this for people's local installs... that'd be great.
